pkg/agent/openflow/pipeline.go

```diff
@@ -115,21 +106,15 @@ var (
 		{sessionAffinityTable, "SessionAffinity"},
 		{serviceLBTable, "ServiceLB"},
 		{endpointDNATTable, "EndpointDNAT"},
-		{EmergencyEgressRuleTable, "CNPEmergencyEgressRule"},
-		{SecurityOpsEgressRuleTable, "CNPSecurityOpsEgressRule"},
-		{NetworkOpsEgressRuleTable, "CNPNetworkOpsEgressRule"},
-		{PlatformEgressRuleTable, "CNPPlatformEgressRule"},
-		{ApplicationEgressRuleTable, "CNPApplicationEgressRule"},
+		{MultiTierEgressRuleTable, "CNPMultiTierEgressRule"},
```

Comment:
shall we rename these tables to APXyz... since they will accommodate both antrea np CRDs

Reply:
Done